lib/github_api/params_hash.rb

```diff
@@ -9,7 +9,7 @@ class ParamsHash < DelegateClass(Hash)
     include Normalizer
     include MimeType
 
-    REQUEST_PARAMS = [:accept, :media, :data, :raw, :content_type, :headers]
+    REQUEST_PARAMS = [:accept, :media, :data, :raw, :content_type, :headers, :options]
```

Comment:
I think the `:headers` key should do it for you, e.i. include the header information in the actual request body when the request is fired - [see here](https://github.com/piotrmurach/github/blob/master/lib/github_api/connection.rb#L38) . That was one of the reasons I've added it originally to allow for preview apis that have different accept headers. Also if you want to see more about the request being fired use `DEBUG=true` when running the actual test or live request.